packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -501,11 +473,27 @@ abstract contract OPContractsManagerBase {
     )
         internal
     {
-        if (!isDevFeatureEnabled(DevFeatures.DEPLOY_V2_DISPUTE_GAMES)) {
+        if (!isSuperGameVariant(_gameType) && !isDevFeatureEnabled(DevFeatures.DEPLOY_V2_DISPUTE_GAMES)) {
             revert OPContractsManager.InvalidDevFeatureAccess(DevFeatures.DEPLOY_V2_DISPUTE_GAMES);
         }
         _dgf.setImplementation(_gameType, _newGame, _gameArgs);
     }
+
+    /// @notice Returns true iff the game type is CANNON or SUPER_CANNON
+    function isCannonOrSuperCannonGameType(GameType _gameType) internal pure returns (bool) {
```

Comment:
I kind of want to call this `isOpProgramVariant` but to be consistent we should probably make it `isCannonVariant` because contracts always call op-program+cannon just cannon.  It's a bit ambiguous though..

Happy for you to ignore these naming suggestions and go with whatever you think.